checks/zookeeper.go

```diff
@@ -55,3 +65,65 @@ func (chk ZooKeeperRUOK) Status() (int, string, error) {
 	}
 	return 1, "Failed: " + failed, nil
 }
+
+type ZooKeeperServerStats struct {
+	timeout time.Duration
+	servers []string
+    minLatency int64
+    maxLatency int64
+    avgLatency int64
+}
+
```

Comment:
Can we get a documentation comment here? Should specify units for the latencies.